### DISCUSSION TITLE
I've finished the tutorial and have done a project of my own that is working fine. What's next?

### DISCUSSION THREAD

**Discussion by NarithChesda** [smartcontractkit/full-blockchain-solidity-course-py#1435] [Q&A] [answered]
I've build some simple project but what I found out is that, there are a lot more that I don't know about blockchain, defi, nfts gaming etc.. What should I do next to get myself educate?

**Comment by cromewar** [comment, marked as answer]
As Patrick would say you are now forbidden to take another tutorial!! Now is time to build, and interesting enough Chainlink Hackathon is about to start: Register yourself [here](https://chain.link/hackathon) and build a project ;)
This is the best way to improve as a developer.
